content/blog/en/version-7-dojo-widgets.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Announcing Dojo 7 Widgets
+date: 2020-04-20T08:00:00.000Z
+author: Eric Osmundson
+---
+
+Since the last Dojo release, we have been expanding the features and improving the developer experience of the [Dojo Widgets](https://github.com/dojo/widgets) library. With new advances in Dojo features, functionality, and recommended best practices, Dojo version 7 provides substantial improvements to make it easier for developers to build efficient and modern web applications.
+
```

Comment:
Could you add `<!-- more -->` here please? And I have requested a featured image for the blog that can go in once we have it

Reply:
Images updated, including featured image.